modules/zero-trust-manager-configure-vault.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+//
+// * security/zero_trust_workload_identity_manager/zero-trust-manager.adoc
```

Comment:
There are a lot of assumptions that the Vault server has been configured a certain way. Is there any other guidance that we will be providing for our customers?